docpages/example_programs/misc/using-emojis.md

```diff
@@ -0,0 +1,29 @@
+\page using-emojis Using Emojis
+
+Need your bot to use an emoji? Then you've come to the right place! Here are three examples of using emojis.
+
+\note If your bot isn't in the guild where you want to use the custom emoji from, it won't work, giving you `dpp::exception_error_code::err_unknown_emoji`.
```

Comment:
Please take `dpp::exception_error_code::err_unknown_emoji` out of the code-block so it links correctly (allowing people to click it to view it)

Reply:
Also the code block is not the problem

Reply:
True, not sure why your branch isn’t showing it linking. Either way, we’ve stated this several time and I’ll update the docs convention for it, you do not put things like dpp::cluster and whatnot in code blocks as we all agreed it looks awful when linked inside a code block.